docs/hosting-a-node.md

```diff
@@ -101,11 +101,29 @@ _As of 22nd April 2024, a full node without logs uses **~100 GB** of disk space.
 Telemetry plays a critical role in monitoring and managing blockchain nodes efficiently. 
 Below is an overview of how metrics is integrated and utilized within our node systems.
 
-Metrics is enabled in nodes by default. It's possible to disable it by setting  `--enable-metrics=false`.
-By default, a [prometheus](https://prometheus.io/docs/introduction/overview/) server is available at `localhost:2112/metrics` with the metrics.
+Metrics is not enabled in nodes by default. It's possible to enable it by setting  `--enable-metrics`.
```

Comment:
just wonder if it should be is or are

Reply:
I would guess its the metrics server so thats why it was is?

Reply:
probably question to @nwbrettski or someone else with better english understanding than mine 😆 just it was not feeling right for me, thats why wanted to double check

Reply:
Suggest modifying it to be

Metrics are not enabled in nodes by default. You can enable them by setting --enable-metrics.